eng/build.yml

```diff
@@ -69,36 +69,42 @@ parameters:
   default: false
 
 jobs:
-- template: common/templates/job/job.yml
+- template: common/templates-official/job/job.yml
```

Comment:
@self?

Reply:
`@self` is only needed when you're providing a path as a parameter for a file in another repo, such as the 1ES template. Here, this file is in this repo and the path is in the repo, so it doesn't need it.